GUI/Controls/ManageMods.cs

```diff
@@ -1246,6 +1246,15 @@ private int CompareRows(DataGridViewRow a, DataGridViewRow b)
             return CompareColumn(a, b, ModName);
         }
 
+        /// <summary>
+        /// Compare two rows based on one of their columns
+        /// </summary>
+        /// <param name="a">First row</param>
+        /// <param name="b">Second row</param>
+        /// <param name="col">The column to compare</param>
+        /// <returns>
+        /// -1 if a&lt;b, 1 if b&gt;a, 0 if a==b
```

Comment:
```suggestion
        /// -1 if a&lt;b, 1 if b&lt;a, 0 if a==b
```

...I think

Reply:
Oops, yes, was only supposed to swap one of the operator or the operands ordering...